src/test/java/org/spectrumauctions/sats/opt/examples/CustomizedExamples.java

```diff
@@ -1,6 +1,7 @@
 package org.spectrumauctions.sats.opt.examples;
 
 import com.google.common.collect.ImmutableMap;
+import org.junit.Ignore;
```

Comment:
My reasoning was that they take too long for stuff that's covered by other tests anyway. But yeah, I may want to have that ensured anyway, since people can still skip the tests when they first install sats-opt... Well but I think I'll leave that ignored for now, if that's ok for you?